readme.md

```diff
@@ -1,17 +1,21 @@
-# Service Fabric Cluster with Datadog installed as a VM extension on each node
+# Datadog Service Fabric Cluster Template
 
-These ARM templates set up unsecure Service Fabric clusters that include the Datadog agent on each node as a VM extension.
+These ARM templates set up an insecure Service Fabric cluster that includes the Datadog Agent on each node as a VM extension.
```

Comment:
Why is it insecure? It sounds bad 🤔

Reply:
For it to be secure ([relevant docs](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-security)) you have to have a custom domain name for accessing the cluster and provide your own certificate for that domain (you can also use Kerberos but it is Windows only and deprecated). These are things that you can't (fully) do with the ARM template, so at most I think we can point users to Microsoft docs on how to secure a cluster and recommend users of the template to do it.